openapi/components/schemas/KycRequestDocuments.yaml

```
@@ -0,0 +1,25 @@
+description: Documents to be requested from customer.
+type: array
+items:
+  type: object
+  required:
+    - type
+  properties:
+    type:
+      description: Document type to be requested from customer.
+      allOf:
+        - $ref: ./KycDocument/KycDocumentTypes.yaml
+    subtypes:
+      description: Document subtype allowed for customer.
```

suggestion: I'm not sure if we need to state "for the customer here"

I feel that it is already described

```suggestion
      description: Allowed document subtype.
```